tests/test_file.py

```diff
@@ -8,19 +8,19 @@
 def test_version():
     from corsikaio import CorsikaFile
 
-    with CorsikaFile('tests/resources/mmcs65') as f:
+    with CorsikaFile('../tests/resources/mmcs65') as f:
```

Comment:
Please do not change the paths here.

Just run `pytest` from the base directory